gems/aws-sdk-core/spec/fixtures/schema/validates_using_the_schema/api.json

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
Why are these being deleted, do we not use them anymore?

Reply:
They're not.